tools/extra_path.sh

```diff
@@ -13,6 +13,7 @@ else
     TOOL_DIR="$(pwd)"
 fi
 
+export PATH="${TOOL_DIR}":"${PATH:-}"
```

Comment:
If we do not use jq, we may not need it.

Reply:
I see, understand